harness/tests/integration_cases/test_raft.rs

```diff
@@ -5075,3 +5075,174 @@ fn test_read_when_quorum_becomes_less() {
         .unwrap();
     assert!(!network.peers[&1].read_states.is_empty());
 }
+
+#[test]
+fn test_uncommitted_entries_size_limit() {
```

Comment:
How about just follow the test case `TestUncommittedEntryLimit` in etcd for better maintaining in the future?